test/bash/tools/flux.bats

```diff
@@ -40,10 +44,6 @@ teardown() {
     rm -rf "${TEST_ROOT_DIR}"
 }
 
-teardown_file () {
-  rm -rf "${BUILDPACK_CACHE_DIR}"
```

Comment:
shouldn't we cleanup cache dir, if we use a temporary folder? wouldn't it otherwise create new folders on every run on local dev machine?

Reply:
indeed, will add that